easybuild/easyconfigs/s/synthcity/synthcity-0.2.10-foss-2022b.eb

```
@@ -0,0 +1,239 @@
+    }),
+    ('pybcj', '1.0.2', {
+        'modulename': 'bcj',
+        'checksums': ['c7f5bef7f47723c53420e377bc64d2553843bee8bcac5f0ad076ab1524780018'],
+    }),
+    ('pycryptodomex', '3.20.0', {
+        'modulename': 'Crypto',
+        'checksums': ['7a710b79baddd65b806402e14766c721aee8fb83381769c27920f26476276c1e'],
+    }),
+    ('pyppmd', '1.0.0', {
+        'checksums': ['075c9bd297e3b0a87dd7aeabca7fee668218acbe69ecc1c6511064558de8840f'],
+    }),
+    ('pyzstd', '0.16.0', {
+        'checksums': ['fd43a0ae38ae15223fb1057729001829c3336e90f4acf04cf12ebdec33346658'],
+    }),
+    ('texttable', '1.7.0', {
+        'checksums': ['2d2068fb55115807d3ac77a4ca68fa48803e84ebb0ee2340f858107a36522638'],
+    }),
+    ('py7zr', '0.20.6', {
+        'checksums': ['d036dee11fce69ad8d4fa86025ccfc4a3511ec27ee1c6b5bd8d6759313dbd077'],
+    }),
+    ('torchtuples', '0.2.2', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['186625230a149cc09f64116d51b203ffefe78160f5a0445adad195893663f55b'],
+    }),
+    ('pycox', '0.2.3', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['9ea3c64a4a650ccf6c96cf512712de330f2d75de32122d86995c7cd37ff105d1'],
+    }),
+    ('keopscore', '2.2.3', {
+        'checksums': ['64d5dad1e8c806d7070cdc60e48fd5bbf006f2bf1afd39ad3fa5e9fb213517d2'],
+    }),
+    ('pykeops', '2.2.3', {
+        'checksums': ['2e2cba1de5e05c35559957a14f41cb5165dc667cc51b3b2118d7e0027eb435a1'],
+    }),
+    ('pyts', '0.13.0', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['b49608267b686ea693dba31316ef2b22ad73ea29b27144696c347809ecd5ad62'],
+    }),
+    ('tsai', '0.3.9', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['0e6460e9ff075176c8af25ef90444ef2e287bc5fe5510e2a0f427e4d0b37ebc0'],
+    }),
+    ('accelerate', '0.31.0', {
+        'checksums': ['b5199865b26106ccf9205acacbe8e4b3b428ad585e7c472d6a46f6fb75b6c176'],
+    }),
+    ('be_great', '0.0.7', {
+        'source_tmpl': '%(name)s-%(version)s-py3-none-any.whl',
+        'checksums': ['824f10482581f5211d3a435d8b14adcc6c21b38d952f00b6a792fc7c089788b1'],
+    }),
+    ('xgbse', '0.2.3', {
+        'preinstallopts': local_xgbse_preinstallopts,
+        'source_urls': ['https://github.com/loft-br/xgboost-survival-embeddings/archive/'],
+        'sources': [{'download_filename': 'v%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}],
+        'checksums': ['9e6b71539b2b533c00eddfe7681cdd541f393d9573594eca358d263b141c127d'],
+    }),
+    ('arfpy', '0.1.1', {
+        'checksums': ['88170d5e72638b0dbfec28cfbdfee02e97bd6a06d5a636e960acd5d90d480707'],
+    }),
+    ('numpy', '1.23.5', {
+        'patches': [
+            {'name': 'numpy-1.20.3_fix-fortran-compiler-error.patch', 'alt_location': 'Scipy-bundle'},
+            {'name': 'numpy-1.22.3_disable-broken-override-test.patch', 'alt_location': 'Scipy-bundle'},
```

@pavelToman The `alt_location` is case-sensitive, and `SciPy-bundle` has a capital `P` in it:

```suggestion
            {'name': 'numpy-1.20.3_fix-fortran-compiler-error.patch', 'alt_location': 'SciPy-bundle'},
            {'name': 'numpy-1.22.3_disable-broken-override-test.patch', 'alt_location': 'SciPy-bundle'},